Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3.6
```

Comment:
We have already an image that has the k8s cli utilities: https://github.com/Financial-Times/k8s-cli-utils.
It is used by Jenkins to communicate with the clusters.
It is a little out of date, but maybe we can use it.